helpers/pwa.js

```diff
@@ -9,8 +9,10 @@ const generateIconsContentForManifest = (savedImages, manifestJsonPath) => {
     )
     .map(({ path, width, height }) => ({
       src: manifestJsonPath
-        ? file.getRelativeFilePath(manifestJsonPath, path)
-        : path,
+        ? file.convertBackslashPathToSlashPath(
```

Comment:
I think it's time for a refactor here as this has been more complicated to follow as part of ternary expression and feels like we start to repeat some code. I propose introducing a generic function in file.js service, such as;

```javascript
export const getRelativeImagePath = (outputFilePath, imagePath) => {
    if (outputFilePath) {
        return convertBackslashPathToSlashPath(getRelativeFilePath(outputFilePath, imagePath));
    }
    return convertBackslashPathToSlashPath(imagePath);
}
```

Then we can simply call this function in 3 places where we need this functionality.

Reply:
Okay makes sense. Should I do this refactor and push, or you can make the change directly?

Reply:
Please do so, it's your PR :)

Reply:
Ok on it!